### DISCUSSION TITLE
Could JuiceFS connect to multiple object storage at the same time?

### DISCUSSION THREAD

**Discussion by Clara12062** [juicedata/juicefs#5153] [Q&A] [answered]
Hi
I am new to JuiceFS and I would like to know if the S3 gateway feature of JuiceFS supports connecting to multiple object storage clusters?
 
My use case is as follows: In different namespaces on Kubernetes, there are different Minio tenants. If I want a unified terminal entry point, rather than using different svc domains. Could I use JuiceFS?

**Comment by zhijian-pro** [comment, marked as answer]
Currently, a single gateway cannot export multiple file systems
